torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -15,6 +15,7 @@ namespace rpc {
 struct RpcBackendOptions {
   RpcBackendOptions() noexcept = default;
```

Comment:
There is an relevant compiling error.

Try removing `noexcept` ? 

I guess it's a mismatch with std::string's default constructor.